robots/flakefinder/main.go

```diff
@@ -44,31 +44,32 @@ func flagOptions() options {
 	flag.DurationVar(&o.merged, "merged", 24*7*time.Hour, "Filter to issues merged in the time window")
 	flag.Var(&o.endpoint, "endpoint", "GitHub's API endpoint")
 	flag.StringVar(&o.token, "token", "", "Path to github token")
-	flag.BoolVar(&o.isPreview, "preview", false, "if report should be written to preview directory")
+	flag.BoolVar(&o.isPreview, "preview", false, "Whether report should be written to preview directory")
+	flag.StringVar(&o.prBaseBranch, "pr_base_branch", PRBaseBranchDefault, fmt.Sprintf("Base branch for the PRs (default: '%s')", PRBaseBranchDefault))
+	flag.StringVar(&o.reportOutputChildPath, "report_output_child_path", "", fmt.Sprintf("Child path below the main reporting directory '%s' (i.e. 'master', default is '')", ReportsPath))
```

Comment:
Would it make sense to just set this to the prBaseBranch? Maybe with the exception of "" for the master branch?

Reply:
I wanted it to be backwards compatible at first, so I left it with the empty path, which will point towards the same path where the reports are generated currently. This makes sense IMHO because we already have communicated the base URL where to fetch the reports from.

We might consider moving the directories at a later time and then just change the settings for the jobs.

Reply:
I mean, we don't need to change the code, just the configuration for this. That was what I had in mind.

Reply:
ok